charts/ocis/values.yaml

```diff
@@ -216,8 +216,10 @@ features:
       minSpecialCharacters: 0
       # the minimum amount of digits the password needs to have
       minDigits: 0
-      # the path to the file containing the list of banned passwords
-      bannedPasswordsList: ""
+      # list of banned passwords
+      bannedPasswords: []
```

Comment:
Not sure if this will be sufficient in the future. I heard that there will be thousands of passwords in this banned password list. But we can iterate over this later

Reply:
I don't know where this should go. If we have thousands of passwords we might hit the ConfigMap size limit quite quickly.

But to be honest: is a list of thousands of passwords that is regularly updated still application configuration? I would rather thread this like eg. ClamAV virus signature databases... That need to be fetch from some sort of mirror